src/dropdown/dropdown.js

```diff
@@ -169,7 +188,13 @@ angular.module('ui.bootstrap.dropdown', ['ui.bootstrap.position'])
       if ( !dropdownCtrl ) {
         return;
       }
-      dropdownCtrl.dropdownMenu = element;
+      var tplUrl = attrs.templateUrl;
+      if ( tplUrl ) {
```

Comment:
Remove the spaces around `tplUrl` - also it would be nice if you could remove the spaces around `!dropdownCtrl` above as well.